### PR TITLE
[chore] add queue benchmarks

### PR DESCRIPTION
**Pull request by atoulme**

### CODE REVIEW

File: exporter/exporterhelper/internal/bounded_memory_queue_test.go [hunk 1]
```diff
@@ -150,6 +151,62 @@ func TestShutdownWhileNotEmpty(t *testing.T) {
...
+}
+func Benchmark_QueueUsage_10000_5_250000(b *testing.B) {
+	queueUsage(b, 10000, 5, 250000)
+}
+func Benchmark_QueueUsage_10000_10_250000(b *testing.B) {
+	queueUsage(b, 10000, 10, 250000)
+}
+
+func queueUsage(b *testing.B, capacity int, numConsumers int, numberOfItems int) {
+	for i := 0; i < b.N; i++ {
```
Comment (dmitryax) [inline exporter/exporterhelper/internal/bounded_memory_queue_test.go:197]: Please add memory allocation reporting

Comment (atoulme) [reply exporter/exporterhelper/internal/bounded_memory_queue_test.go:197]: Sorry, I'm not sure how that is best done - googling gives me weird results. Any example?

Comment (dmitryax) [reply exporter/exporterhelper/internal/bounded_memory_queue_test.go:197]: Just add `b.ReportAllocs()`

Comment (atoulme) [reply exporter/exporterhelper/internal/bounded_memory_queue_test.go:198]: Done, thanks!

**Review by dmitryax — APPROVED** [review]

**Pull request merged**